packages/notifi-react/README.md

```diff
@@ -1,81 +1,139 @@
 # Notifi React SDK
 
-`@notifi-network/notifi-react` accelerates the development of Notifi integrated dapps by providing:
+`@notifi-network/notifi-react` accelerates the development of Notifi integrated dapps.
 
-- Out-of-the-box React components (`NotifiCardModal`) by which developers can just plug and play.
+-  [Out of The Box Implementation](#out-of-the-box-implementation)
+-  [Customized Implementation](#advanced-customized-implementation)
+-  [Creating Your Own Card](#creating-your-own-card)
+-  [Migrating from an older version of the SDK ](#migrate-your-card)
 
-- React Contexts for more advanced use cases in case developers want to build their own UI components.
 
-> Give it a hand-on by checking out the [example project](https://notifi-sdk-ts-vercel-notifi-react-example-v2.vercel.app/)
 
-## React Components (NotifiCardModal)
+> Play around with a Card [Example](https://notifi-sdk-ts-vercel-notifi-react-example-v2.vercel.app/)
```

Comment:
The official link is now [notifi-sdk-ts-notifi-dapp-example.vercel.app](https://notifi-sdk-ts-notifi-dapp-example.vercel.app/)

Reply:
fixed